artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/queue/UpdateQueue.java

```diff
@@ -31,7 +31,7 @@
    String name;
 
    @Option(name = "--deleteOnNoConsumers", description = "whether to delete when it's last consumers disconnects)")
-   Boolean deleteOnNoConsumers = null;
+   String deleteOnNoConsumers = null;
```

Comment:
I need a parameter with 3 possible values:
- null: no update to the queue property will happen
- true: the queue property will be updated using a value of true
- false: the queue property will be updated using a value of false

I know that is something different, but it's a different use case..

In the original version of UpdateQueue I've declared it as Boolean, but after creating a queue with 
deleteOnNoConsumers=true there are no ways to use an UpdateQueue command to turn deleteOnNoConsumers to false.